Feed: match/request buttons don't work

No visible effect when clicking one of the buttons:

![image](https://user-images.githubusercontent.com/551624/27473453-98530510-57ff-11e7-83ea-e20f58b6ccee.png)

js console:
```

angular.js:5209 Error: Could not resolve 'postRequests' from state 'feed'
    at Object.y.transitionTo (angular-ui-router.js:1459)
    at Object.y.go (angular-ui-router.js:1421)
    at angular-ui-router.js:1976
    at angular.js:8208
    at e (angular.js:2434)
    at angular.js:2563
```
and 
```

angular.js:5209 Error: Could not resolve 'postRequests' from state 'feed'
    at Object.y.transitionTo (angular-ui-router.js:1459)
    at Object.y.go (angular-ui-router.js:1421)
    at angular-ui-router.js:1976
    at angular.js:8208
    at e (angular.js:2434)
    at angular.js:2563
```

--- a/webofneeds/won-owner-webapp/src/main/webapp/app/components/feed-item.js
+++ b/webofneeds/won-owner-webapp/src/main/webapp/app/components/feed-item.js
@@ -86,7 +86,7 @@ function genComponentConf() {
         <div class="fi__footer" ng-show="self.unreadMatchesCount() || self.unreadRequestsCount()">
             <div class="fi__footer__indicators">
                 <a class="fi__footer__indicators__item clickable"
-                   ui-sref="postMatches({myUri: self.ownNeed.get('@id')})"
+                   ui-sref="post({connectionType: self.WON.Suggested, openConversation: null, connectionUri: null, postUri: self.needUri})"
                    ng-show="self.unreadMatchesCount()">
                     <img src="generated/icon-sprite.svg#ico36_match" class="fi__footer__indicators__item__icon"/>
                     <span class="fi__footer__indicators__item__caption">
@@ -95,7 +95,7 @@ function genComponentConf() {
                     </span>
                 </a>
                 <a class="fi__footer__indicators__item clickable"
-                   ui-sref="postRequests({myUri: self.ownNeed.get('@id')})"
+                   ui-sref="post({connectionType: self.WON.RequestReceived, openConversation: null, connectionUri: null, postUri: self.needUri})"
                    ng-show="self.unreadRequestsCount()">
                     <img src="generated/icon-sprite.svg#ico36_incoming" class="fi__footer__indicators__item__icon"/>
                     <span class="fi__footer__indicators__item__caption">
@@ -141,6 +141,7 @@ function genComponentConf() {
                     ownNeed,
                     ownNeedContent,
                     connections,
+                    WON: won.WON,
                     nrOfConnections: connections? connections.size : 0,
                     createdOn: ownNeed && relativeTime(lastUpdated, ownNeed.get('dct:created')),
                     unreadCounts: unreadCountsByNeedAndType && unreadCountsByNeedAndType.get(self.needUri),

